public/app/plugins/datasource/loki/components/LokiQueryFieldForm.tsx

```diff
@@ -173,8 +182,13 @@ export class LokiQueryFieldForm extends React.PureComponent<LokiQueryFieldFormPr
               syntaxLoaded={syntaxLoaded}
             />
           </div>
+          {children}
```

Comment:
Nitpick: Mentioned it also in some other PR, I think it would be clearer to use some other prop for this like `extraFieldEl` or something like that as the field is not intuitively a wrapper element so using children seem bit weird here.

Reply:
Fixed.